views/events/performance_events.view.lkml

```diff
@@ -254,6 +254,37 @@ view_label: " Performance Events"
   }
 
 
+  dimension: numOfRequest {
```

Comment:
If you want to see these under the measures, this should be a measure
```suggestion
  measure: numOfRequest {
```

Reply:
Heres an example of how measures work, duration exists as a dimension, using `${TABLE}.duration`, once its in Looker as a dimension we can reference it in a measure, using the name of the dimension, instead of `${TABLE}.duration`
```
  dimension: duration {
    label: "Duration"
    description: "The Duration of the time taken for the performance event to load in milliseconds."
    type: number
    sql: ${TABLE}.duration ;;
  }
```
```
measure: duration_median {
    group_label: "Duration Measures"
    label: "Duration (Med)"
    description: "The median Duration across all instances within the grouping."
    type: median
    sql: ${duration} ;;
    value_format_name: decimal_1
  }
```

Reply:
so in your case we'll need to add 4 more fields, as measures, based on the type of analysis (sum, avg, median etc.) you want to do